paket.dependencies

```diff
@@ -1,8 +1,8 @@
 version 5.198.0
 source https://api.nuget.org/v3/index.json
-frameworks: net46, netcoreapp3.1, netstandard2.0
+frameworks: netcoreapp3.1, netstandard2.0
 
-nuget FSharp.Core 4.3.4
+nuget FSharp.Core 5.0.0
```

Comment:
The version is locked, because we should not update it - https://github.com/dotnet/fsharp/blob/main/docs/fsharp-core-notes.md#libraries-should-target-lower-versions-of-fsharpcore